City-of-Pori/pori-kada
# PORI-311 Changed the some blocks to resemble Turku.fi style

To test:

1. drush fra -y && drush cc all
2. Go to see social media posts on front page and make sure that there is now icon of the social media platform on each post, that the icon links to the original post and that the post date is next to the social media icon.
